src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -467,11 +467,12 @@ public SnapShot getSnapShot(@Nonnull ProjectId projectId) throws LoginTimeoutExc
 		ClientRequestException {
 		String requestUrl = PROJECT_SNAPSHOT + "?projectid=" + projectId.get();
 		Response response = get(requestUrl); // send request to server
-		return retrieveDocumentSnapshotFromServerResponse(response);
+		return retrieveDocumentSnapshotFromServerResponse(response, projectId);
 	}
 
-	private SnapShot retrieveDocumentSnapshotFromServerResponse(Response response)
+	private SnapShot retrieveDocumentSnapshotFromServerResponse(Response response, @Nonnull ProjectId projectId)
```

Comment:
this seems odd, it's declared nonnull then there's a check for a null, and either way the variable is never used?